\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage{xcolor}
\usepackage[numbers, compress]{natbib}
\bibliographystyle{unsrtnat}
\usepackage{chapterbib}
\usepackage[a4paper, portrait, margin=2cm]{geometry}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{hyperref}
\usepackage[toc,page]{appendix}

\newcommand{\igjindent}{\,\,\,\,\,\,\,\,\,}
\newcommand{\be}{\begin{equation}}
\newcommand{\ee}{\end{equation}}

\title{Mitochondrial network structure shapes cell-to-cell mtDNA variability generated by cell divisions}
\author{Robert C. Glastad${}^1$, Iain G. Johnston${}^{1,2,*}$ \\ \footnotesize ${}^1$ Department of Mathematics, University of Bergen, Bergen, Norway \\ \footnotesize ${}^2$ Computational Biology Unit, University of Bergen, Bergen, Norway \\ \footnotesize ${}^*$ correspondence to \url{iain.johnston@uib.no}}
\date{}

\begin{document}

\maketitle
\begin{abstract}
Mitochondria are highly dynamic organelles, containing vital populations of mitochondrial DNA (mtDNA) distributed throughout the cell. Mitochondria form diverse physical structures in different cells, from cell-wide reticulated networks to fragmented individual organelles. These physical structures are known to influence the genetic makeup of mtDNA populations between cell divisions, but their influence on the inheritance of mtDNA at divisions remains less understood. Here, we use statistical and computational models to show how mitochondrial network structures shape the genetic and physical inheritance of mtDNA populations at cell divisions. We compare different models for the arrangement and diffusion of mtDNA content inside and outside the reticulated network and quantify the effect on mtDNA heteroplasmy and copy number across this range of cases. We assess the use of moment-based approximations to describe heteroplasmy variance and identify several cases where such an approach has shortcomings. We show that biased inclusion of one mtDNA type in the network can substantially increase heteroplasmy variance (acting as a genetic bottleneck), and controlled distribution of network mass and mtDNA through the cell can conversely reduce heteroplasmy variance below a binomial inheritance picture. Network structure also allows the generation of heteroplasmy variance while controlling copy number inheritance to sub-binomial levels, reconciling several observations from the experimental literature. 
\end{abstract}


\section{Introduction}
Mitochondria are vital bioenergetic organelles responsible for the majority of cellular energy production in eukaryotes \citep{spinelli2018multifaceted, smith2015mitochondrial}. Due to their evolutionary history, mitochondria have retained small genomes \citep{martin1998gene, gray2001origin, roger2017origin} that encode genes central to their energy production \citep{johnston2016evolutionary, allen2017corr}. MtDNA in several taxa, including many animals, is subject to a high mutation rate relative to the nucleus, and mutations in mtDNA cause cellular dysfunction, and are involved in a range of human diseases \citep{wallace2013mitochondrial, burr2018mitochondrial}. As mtDNA is predominantly uniparentally transmitted \citep{greiner2015most}, the question arises of how eukaryotes avoid the gradual accumulation of mtDNA mutations, known as Muller's ratchet \citep{muller1964relation}. 

The proportion of mutant mtDNA in a cell is usually referred to as heteroplasmy, and diseases are often manifest when heteroplasmy exceeds a certain level \citep{rossignol2003mitochondrial}. Eukaryotes may deploy a combination of strategies to generate cell-to-cell variability in inherited heteroplasmy \citep{stewart2015dynamics, radzvilavicius2017mitigating, johnston2019varied, edwards2021avoiding}, thus potentially generating offspring with heteroplasmies below the threshold \citep{burr2018mitochondrial, burgstaller2014mtdna, segui2009mitochondrial}. For instance, in mammalian development, a developing female produces a set of oocytes for the next generation. Through an effective `genetic bottleneck', oocytes with different heteroplasmies are generated \citep{johnston2019varied}. This range of heteroplasmies means that some cells may inherit a lower heteroplasmy than the mother's average. Across species, in concert with selection \citep{fan2008mouse, stewart2008strong, hill2014selective, floros2018segregation, lieber2019mitochondrial, jakubke2021cristae}, this generation of variation allows shifts in heteroplasmy between generations \citep{burgstaller2018large, wei2019germline}.

%In mammals, cell-to-cell variability from this effective bottleneck (increasing heteroplasmy variance) and purifying selection \citep{fan2008mouse, stewart2008strong, hill2014selective, floros2018segregation, lieber2019mitochondrial} (changing heteroplasmy mean) act to facilitate heteroplasmy shifts between generations 

The mtDNA bottleneck has been suggested to incorporate a number of different mechanisms \citep{johnston2015stochastic}. These include mtDNA depletion \citep{cree2008reduction, cao2007mitochondrial, cao2009new}, and subpopulation replication \citep{cao2007mitochondrial, wai2008mitochondrial} in mammals, with a potential role for gene conversion in several other taxa \citep{edwards2021avoiding}, all coupled with random effects from partitioning of mtDNA at cell divisions. This partitioning is a focus of this report. Generally, when a cell divides, its mtDNA population will be partitioned between its daughter cells. Any deviation from precise deterministic partitioning (exactly half the mtDNA molecules of every genetic type go into each daughter) will likely lead to the daughter cells inheriting different heteroplasmy levels. 

%and strong purifying selections \citep{jakubke2021cristae} in fungi and yeast \citep{mendoza2020mitochondrial}.  %Selection and amplification mechanisms are thought to act to prevent increases in cell heteroplasmy levels \citep{stewart2015dynamics,stewart2008strong}. 

Models for this segregation have typically regarded the cellular mtDNA population as well-mixed, but the mitochondria containing the population have varied morphologies and dynamics in different cell types. Mitochondria can form cell-wide networks in some cell types, undergoing fission and fusion \citep{mendoza2020mitochondrial, sukhorukov2012emergence, hoitzing2015function, zamponi2018mitochondrial, carlton2020membrane}. These dynamic structures, together with mtDNA turnover, are linked to quality control and genetic dynamics of mtDNA populations \citep{jakubke2021cristae,pla2019reshaping, chapman2020maintenance}. In both somatic tissues \citep{mouli2009frequency, tam2013mathematical, patel2013optimal, tam2015context}, or the germline \citep{edwards2021avoiding, lieber2019mitochondrial}, the balance between fusion, fission and selective degradation of individual dysfunctional mitochondria has emerged as an important influence on mtDNA populations \citep{mouli2009frequency,patel2013optimal,twig2008fission, twig2008mitochondrial, figge2012deceleration}. 

Quantitative progress modelling the spatial influence of these mitochondrial dynamics on mtDNA quality control is advancing \citep{mouli2009frequency, tam2013mathematical, patel2013optimal, tam2015context,figge2012deceleration}. In particular, the role of network structure in generating cell-to-cell mtDNA variability via modulating mtDNA turnover has been addressed with recent stochastic modelling \citep{edwards2021avoiding, aryaman2019mitochondrial}. \textcolor{red}{These studies report that cell-to-cell mtDNA variability increases with the proportion of fragmented mitochondria in the cell, the rate of turnover, and the length of the cell cycle. Moreover, turnover itself may increase due to a highly fragmented network, with a highly fused network effectively masking mitochondria marked for degradation.} However, the influence of mitochondrial network structure on the inheritance of mtDNA during cell divisions remains less studied. Partitioning at cell divisions even in the absence of spatial substructure can constitute an important source of cell-to-cell variability \citep{huh2011non, johnston2015closed}. In yeast, mtDNA inheritance occurs with finer-than-random (binomial) control over the number of mitochondrial nucleoids \citep{jajoo2016accurate}, suggesting that physical mechanisms must exist to exert this control. As mtDNA resides in nucleoids that are physically distributed -- potentially heterogeneously -- throughout the mitochondria of the cell \citep{ilamathi2021new}, we set out to explore how different physical structures of mitochondria may shape mtDNA inheritance. \textcolor{red}{Prior to cell divison, the mitochondrial network may fragment, allowing individual mitochondria to diffuse, or subjecting the population to active randomization\citep{moore2021actin}, or it may remain in its reticulated state, significantly influencing the distribution of mtDNAs in the parent cell \citep{mendoza2020mitochondrial}, ultimately reflecting in daughter cell statistics.}

\section{Results}
\subsection{Network inclusion with genetic bias increases cell-to-cell variability}
To build intuition about the influence of network structure on mtDNA inheritance, we first considered a simple computational model for the spatial distribution of mitochondria and mtDNA within a cell (see Methods). This model consists of a random network structure, with a tunably heterogeneous distribution, simulated within a circular cell (the dimensionality of the model does not affect the statistical considerations of partitioning) (see Fig. \ref{fig:plotcell}). The model network structure is not intended to perfectly match the details of real mitochondrial networks, but rather as a general framework to understand spatial substructure in the cell. The mother cell has $N_0$ mtDNAs, a proportion $h$ of which are mutants ($h$ is heteroplasmy). 

To reflect the fact that different mtDNA types may have different propensities to be included in the reticulated network \citep{twig2008fission}, we use $p$ and $q$ to describe the proportions of wildtype and mutant mtDNAs respectively that are contained in the network; the remainder are in fragments in the cytoplasm (Fig. \ref{fig:plotcell} \text{A}, \textbf{E}). Hence, $p > q$ means that wildtype mtDNAs are more likely to be contained in the network and mutant mtDNAs are less likely to be included; $p=q$ means that both types are equally likely to be in a networked state. Network placement can follow various rules (described below) and mtDNA positions may be subsequently perturbed, but we begin with random placement in the network and no subsequent motion prior to division. We then divide the model cell and explore the statistics of mtDNA copy number and heteroplasmy in the daughter cells. We first consider symmetric partitioning, so that the initial cell is physically halved to produce two daughters; we relax this assumption and consider asymmetric partitioning later.

\begin{figure}
\centering
\includegraphics[width=\linewidth]{plotcell.png}
\caption{\textbf{Snapshots of computationally generated networks and mtDNA arrangements with tunable physical and genetic parameters.}  Networks were generated by a elongation and branching process initialized from a number of seed points (s) uniformly distributed on the perimeter of the cell. Small seed numbers (small s; \textbf{A-C}) usually resulted in heterogeneous network structures, with marked differences in network density across the cell; for large seed number (large s; \textbf{E}), networks were uniformly distributed throughout the cell. Wild type (WT; red) and mutant type (MT; blue) mtDNA molecules were distributed into networks according to the proportions $p$ (WT) and $q$ (MT) and whether or not reticulated mtDNA molecules mutually repulse each other (repel) or not, inducing greater than random spacing between molecules. At cell division, daughter cell statistics were recorded, after which the network was fragmented (no parameter) and individual mtDNA molecules were subjected to repeated diffusion with different strengths (large/medium $\lambda$).}\label{fig:plotcell}
\end{figure}

To explore the influence of mtDNA network placement in the parent cell on the mtDNA statistics of the daughter, we varied network inclusion probabilities $p$ and $q$ and observed copy number and heteroplasmy variance in daughter cells after division (Fig. \ref{fig:sim-model-compare}). We found a clear pattern that $V(h)$ takes minimum values when $p=q$, that is when network inclusion probabilities are equal for mutant and wildtype mtDNA. When the two differ, $V(h)$ increases, with highest values occurring when the majority mtDNA type is exclusively contained in the network and the minority type exclusively in the cytoplasm. 

This result may seem counterintuitive at first glance: when the network is highly heterogeneous, it might be expected that including all mtDNAs there would maximise variance. This is true for copy number variance (Fig. \ref{fig:sim-model-compare}), but not for heteroplasmy variance, because including both types equally induces correlation in their inheritance and lowers variance. The maximum $V(h)$ is achieved by embedding the majority type in the high-variance network and having the minority type in the uncorrelated cytoplasm. 

To gain intuition about this picture, we constructed a statistical model for this process (see Methods). Briefly, we consider four state variables describing the mtDNA population of a daughter cell after a mother divides: $W_n, W_c, M_n, M_c$, for the wildtype ($W$) and mutant ($M$) mtDNAs contained in a reticulated mitochondrial network (${}_n$) or in fragmented mitochondrial elements in the cytoplasmy (${}_c$). An additional variable $U$ describes the proportion of network mass inherited by the daughter cell. The mtDNA profile inherited by the daughter then follows

\begin{equation}\label{eq:model}
    \begin{split}
        & W_n \sim \mathrm{Bin}\left(w_n,U\right) \\
        & W_c \sim \mathrm{Bin}\left(w_c,1/2\right)\\ %\frac{1}{2}\right) \\
        & M_n \sim \mathrm{Bin}\left(m_n,U\right) \\
        & M_c \sim \mathrm{Bin}\left(m_c,1/2\right)\\ %\frac{1}{2}\right),
    \end{split}
\end{equation}
where $w_n = p(1-h)N_0, w_c = (1-p)(1-h)N_0, m_n = qhN_0, m_c = (1-q)hN_0$. We model the proportion $U$ of network mass inherited by a daughter with a beta distribution, with variance $V(U)$ allowed to vary to describe different partitioning regimes of network mass. To compare to simulations, we fit the beta distribution parameters to match the simulated inherited network variance. As $W_n$ and $M_n$ are then drawn from the compound distribution that is binomial with a beta-distributed probability, they follow beta-binomial distributions. We are interested in the variables $N = W_n+W_c+M_n+M_c$ and $h = (M_n+M_c)/N$.

We can immediately numerically extract moments for the system through summing over state variables and calculating expectations, for example, 

\begin{equation}\label{eq:vh-estimate-sum}
    E(f(h)) = \sum_{W_c = 0}^{w_c} P(W_c) \sum_{M_c = 0}^{m_c} P(M_c) \int_0^1 dU \sum_{W_n = 0}^{w_n} P(W_n|U) \sum_{M_n = 0}^{m_n} P(W_n|U) f\left(\frac{M_n+M_c}{W_n+W_c+M_n+M_c}\right).
\end{equation}
Figs. \ref{fig:sim-model-compare} and \ref{fig:sim-model-compare-10} demonstrates good correspondence between simulated and statistics from Eq. \ref{eq:vh-estimate-sum}. However, as these large sums do not admit much intuitive analysis, we sought other approaches to learn the forms of pertinent statistics of the inherited mtDNA population.

The mean and variance of inherited number $N$ are readily derived using the laws of iterated expectation and total variance to account for the compound distribution of networked mtDNA (Appendix):
\begin{equation}
    V(N)=\frac{N_0}{4}+\kappa N_0(\kappa N_0-1)V(U)
\end{equation}
where $\kappa = p(1-h)+qh$ denotes the proportion of total mtDNA placed in the network. Hence $V(N)$ experiences an extra, $V(U)$-dependent term in addition to the expected $N_0/4$ result for a purely binomial distribution. This term is quadratic in the proportion $\kappa$ of mtDNA in the network. This expression well captures the results from simulation and Eq. \ref{eq:vh-estimate-sum} (Fig. \ref{fig:sim-model-compare}).

For heteroplasmy variance, as the ratio of random variables, we cannot extract an exact solution and must instead use a Taylor-expanded approximation (see Methods) to obtain 

\be\label{eq:v1h-net}
    V'(h) \simeq V_1'(h)= \frac{1}{N_0} + 4 V(U) \left( h(1-h) (p-q)^2 - (p h + q(1-h))/N_0 \right)
\ee

Eq. \ref{eq:v1h-net} allows some informative analysis. First, we qualitatively see that an additional, $V(U)$-dependent term is introduced compared to the binomial case (which gives $1/N_0$), illustrating the influence of network heterogeneity on heteroplasmy variance. For large $N_0$, this network term is dominated by the first term in brackets in Eq. \ref{eq:v1h-net}, which is quadratic in $(p-q)$, the difference in inclusion probabilities for the different types of mtDNA. For $p\neq q$, the network is genetically biased towards one of the types, to which there is associated an increase in $V'(h)$. For $p=q$, the network is unbiased, with associated prediction

\begin{equation}
V'_1(h) = \frac{1}{N_0} - \frac{4 p V(U)}{N_0}.
\end{equation}
Under this approximate model, the maximum normalised heteroplasmy variance achievable -- when the majority mtDNA type is completely contained in the network and the minority type completely excluded from it -- is given for example by setting $p = 1, q= 0$ for $h \leq 0.5$:

\begin{equation}\label{eq:symm-max-vh}
   V_1'(h)=\frac{1}{N_0} + 4 V(U) \left( h(1-h) - h/N_0 \right).
\end{equation}

However, this picture does not completely capture the simulation and exact results, where we see a small increase in (normalized) heteroplasmy variance for non-biased increases in inclusion probabilities, instead of a decrease. This reflects the approximate nature of the Taylor expansion process used to derive Eq. \ref{eq:v1h-net}; in the Appendix we show that a second-order expansion provides a compensatory diagonal term (Fig. \ref{fig:app-vh-stats}), but in general we will need further terms in the expansion to perfectly match the true behaviour.  In the Appendix we further show that all higher-order moments and covariances of the mtDNA copy numbers are well captured by theory; it is their combination into an estimate for the moments of a ratio (heteroplasmy) that is challenged here. We will observe below that more instances of this system also challenge this Taylor expansion approach, which has been employed previously \citep{johnston2015stochastic, hoitzing2017stochastic} -- generally, accounting for physical structure induces correlations between mtDNA types that are hard to capture with the Taylor expression (see Discussion).
\begin{figure}
\centering
\includegraphics[width=\linewidth]{symm-50-fig-sim-est-taylor.png}
\caption{\textbf{Genetic bias and physical heterogeneity in mitochondrial networks induce cell-to-cell mtDNA variability in symmetric cell divisions.} Normalised heteroplasmy variance $V'(h)$ (left column) and copy number variance $V(N)$ (right column) for mtDNA randomly distributed in networks.  \textbf{A}: simulations for $h=0.1$; \textbf{B}: simulations for $h=0.5$; \textbf{C}: sum over state variables for $h=0.5$; \textbf{D}: first-order Taylor expansion for $h=0.5$. The three columns to a facet give decreasing network heterogeneity, expressed via different seed numbers, $4, 16$ and $64$ (more seed points give a more homogeneous network). In each panel, variances are given for different values of wild and mutant type network inclusion parameters $p$ (horizontal axis) and $q$ (vertical axis). White baseline reflects the null case from the analytic sum without any network inclusion.}\label{fig:sim-model-compare}
\end{figure}

\subsection{Asymmetric cell divisions are a major source of variability}
The above model has assumed that the cell divides symmetrically, with half the cell volume inherited by each daughter. To generalise to asymmetric cell divisions, we next asked how the proportion of inherited cell volume $p_c$ ($p_c = 1/2$ in the symmetric case) influences the mtDNA statistics in daughters. Clearly, the expected copy number will differ if the inherited proportion differs. The expressions above generalise to

\begin{eqnarray}
    V(N) & = &N_0 p_c(1-p_c) +\kappa N_0(\kappa N_0-1)V(U) \\
    V_1'(h)& = &\frac{1-p_c}{p_c} \frac{1}{N_0} + \frac{V(U)}{p_c^2} \left( h(1-h) (p-q)^2 - (p h + q(1-h))/N_0 \right),
\end{eqnarray}

%% 13.03.22 R
with the result that asymmetric cell divisions can generate large increases in cell-to-cell variability for the smaller daughters (Fig. \ref{fig:sim-model-compare-10}). Small number effects are at play here, with a smaller sampling of the initial cell inevitably leading to greater relative variance.  A decrease in $p_c$ from the symmetric case $p_c = 1/2$ to $p_c = 0.1$ results in a near order-of-magnitude increase in the maximum normalised heteroplasmy variance. Asymmetric divisions also further challenge the Taylor expansion approach, with a systematic underestimation of heteroplasmy variance apparent using this approximation (copy number variance remains well captured).

\begin{figure}
\centering
\includegraphics[width=\linewidth]{asymm-0.1-10-fig-sim-est-taylor.png}
\caption{\textbf{Asymmetric cell divisions can induce large cell-to-cell variability in mtDNA quality.} Normalised heteroplasmy variance $V'(h)$ (left column) and copy number variance $V(N)$ (right column) for mtDNA randomly distributed in networks.  The daughter of interest inherits 10\% of the parent cytoplasm.  \textbf{A}: simulations for $h=0.1$; \textbf{B}: simulations for $h=0.5$; \textbf{C}: sum over state variables; \textbf{D}: first-order Taylor expansion). The three columns to a facet give decreasing network heterogeneity, expressed via different seed numbers, $4, 16$ and $64$ (more seed points give a more homogeneous network). In each panel, variances are given for different values of wild and mutant type network inclusion parameters $p$ (horizontal axis) and $q$ (vertical axis). White baseline reflects the null case from the analytic sum without any network inclusion.}\label{fig:sim-model-compare-10}
\end{figure}


\subsection{MtDNA self-avoidance tightens mtDNA copy number control and can reduce heteroplasmy variance}
We next asked whether the variances of copy number $V(N)$ and heteroplasmy $V(h)$ could be reduced below their `null' binomial value through cellular control. To this end, we modelled self-avoidance of mtDNA molecules within the network, reasoning that such controlled arrangement may allow a more even spread of mtDNAs within the network, and correspondingly lower variability. To accomplish this self-avoidance within our model, we enforce a `halo' of exclusion around each mtDNA placed within the network, so that another networked mtDNA cannot be placed within a distance $l$ of an existing one.%The results are shown in Fig. \ref{fig:sim-model-compare-rep}.

Copy number variance $V(N)$ is decreased substantially by self-avoidance (Fig. \ref{fig:sim-model-compare-rep}). In the case of a homogenous network and high proportions of mtDNA network inclusion, this decrease can readily extend below the binomial null model, allowing more faithful than binomial inheritance, as reported in yeast \citep{jajoo2016accurate}. This sub-binomial inheritance requires both an even network distribution and mtDNA self-avoidance, and hence two levels of active cellular control. Under these circumstances, mtDNA molecules are evenly spread throughout the cell volume, and their inheritance approaches a deterministic proportion of the inherited volume fraction $p_c$.

The effect of mtDNA self-avoidance on heteroplasmy variance is more complicated. For highly heterogeneous network distributions, heteroplasmy variance follows the same qualitative pattern as for the non-repulsive case, with higher variances achieved when network inclusion discriminates wildtype and mutant types. However, for homogeneous network distributions, the opposite case becomes true. Here, network inclusion discrimination \emph{lowers} the heteroplasmy variance induced by cell divisions. The heteroplasmy variance induced by cell divisions can even be controlled to sub-binomial levels in the case of self-avoidance, strong discrimination, and a homogeneous network distribution.

Notably, it is possible for the cell to control copy number variance below the binomial limit while also generating heteroplasmy variance, without biased network inclusion (for example, in the $n = 64, h = 0.5$ cases in Fig. \ref{fig:sim-model-compare-rep}), reflecting a potentially beneficial case for implementing a genetic bottleneck without challenging overall mtDNA levels.

Analytic progress is more challenging for this case, but an imperfect statistical model (Fig. \ref{fig:app-sim-model-compare-repel}; see Methods) can begin to capture some of the qualitative behaviour. The model correctly predicts the direction of change of copy number variance for various network structures, and the capacity to control variance below the binomial value, but the magnitudes of predicted variances are more extreme than those observed in simulation. This discrepancy arises because, to retain tractability, the algebraic model imposes an even spread of mtDNAs more strictly than is applied in the simulation (where this imposition is limited for numerical reasons). The range of variance values in the simulation is thus more limited than those that emerge from model predictions, although the trends of behaviour with governing variables are largely consistent.

\begin{figure}
\centering
\includegraphics[width=\linewidth]{symm-50-both-stats-sim-repel.png}
\caption{\textbf{Repulsion between mtDNAs in the network can increase and decrease cell-to-cell variability from cell divisions.} Simulated normalised heteroplasmy variance $V'(h)$ (left column) and copy number variance $V(N)$ (right column) for mtDNAs with mutual repulsion within the network, under symmetric cell divisions. Rows show different values of initial mutant proportion, with $h=0.1$ in the top row and $h=0.5$ in the bottom row. The three columns to a facet give decreasing network heterogeneity, expressed via different seed numbers, $4, 16$ and $64$ (more seed points give a more homogeneous network).  In each panel, variances are given for different values of wild and mutant type network inclusion parameters $p$ (horizontal axis) and $q$ (vertical axis). White baseline reflects the null case from the analytic sum without any network inclusion.}\label{fig:sim-model-compare-rep}
\end{figure}


\subsection{Diffusion of mtDNA relaxes statistics towards their null value}
The mitochondrial network fragments before cell division. This fragmentation gives a time window during which mtDNAs that were previously constrained by the network structure can diffuse away from their initial position. In the limit of infinite diffusion, network structure will be forgotten and the mtDNA population will be randomly and uniformly distributed throughout the cell, leading to binomial inheritance patterns. We next investigated how limited amounts of diffusion away from the initial structure influence the patterns of mtDNA inheritance.

Fig. \ref{fig:diffusion} shows $V'(h)$ for random placement of mtDNA in the network (left column) and repulsive placement of mtDNA in the network (right column) for 4, 16, and 64 seed points (more seed points give a more homogeneous network), respectively, in rows from the top to the bottom. The diffusion strength $\lambda$ is the mean displacement of individual mtDNA molecules as they undergo \textcolor{red}{a random walk, mimicking the findings that mitochondria may undergo a series of directed bursts of motion from active randomization prior to cell division} \citep{moore2021actin}. Our results indicate that directed bursts of motion of the mitochondria indeed work to wash away the effects of the network on daughter cell statistics, resulting in binomial segregation. Notably, there are cases in which extremely heterogeneous mitochondrial networks can leave an imprint on mtDNA inheritance despite high diffusion strength (top row).

\begin{figure}
\centering
\includegraphics[width=\linewidth]{plotlambda.png}
\caption{\textbf{Network fragmentation and diffusion reverse the variance induced by mitochondrial networks.} Normalised heteroplasmy variance $V'(h)$ after repeatedly perturbing mtDNAs with mean displacement $\lambda$ from their original, networked positions. The left column shows $V'(h)$ for random mtDNA placement in the network; the right column shows $V'(h)$ for repulsive mtDNA placement in the network. The three rows show decreasing levels of network heterogeneity, expressed via different seeds numbers (more seed points give a more homogeneous network). As the diffusion strength increases, effects on cell statistics due to the network is washed away regardless of the underlying mtDNA distribution model.}\label{fig:diffusion}
\end{figure}

\section{Discussion}
%Mitochondrial dynamics are known to produce a number of beneficial outcomes for individual cells. Starting with the question of how these processes could influence the segregation of mitochondria, we asked whether these processes also could contribute to the mitochondrial `bottleneck', or segregation of mutant load.

We have demonstrated that mitochondrial network structure can control the variability due to cell divisions in both mtDNA copy number and heteroplasmy inheritance, in both directions. When different mtDNA genotypes have different propensities for network inclusion, random arrangement in a heterogeneous network generates (much) more variability than could be achieved through random cytoplasmic arrangement alone. On the other hand, ordered arrangement in a homogeneous network can control heteroplasmy variance below the binomial level expected from random partitioning. In concert, homogeneous network structure dramatically reduces copy number variance to sub-binomial levels (as observed experimentally in Ref. \citep{jajoo2016accurate}), and heterogeneous network structure correspondingly increases it. Notably, it is possible to tightly control copy number while generating heteroplasmy variance -- a strategy that may be useful in implementing a beneficial genetic bottleneck to segregate mtDNA damage.

How do mitochondria in different taxa and tissues correspond to the different regimes in our model? In animals, mitochondrial structure is highly varied, from largely fragmented organelles (low $p$ and $q$ in our model) to highly reticulated networks (high $p$ and/or $q$), with mtDNA nucleoids moving freely throughout the mitochondrial reticulum, leading to a random distribution of mtDNA within the network \citep{legros2004organization, sasaki2017live}. Animal mitochondria are expected to fragment prior to cell divisions, with a spectrum of active randomization mechanisms \citep{pangou2021multifaceted, moore2021actin}, captured by the diffusion behaviour in our model. Fungal mitochondria, on the other hand, are often inherited without fragmenting the network, which remains in its reticulated state through cell division \citep{mendoza2020mitochondrial}. \textit{Saccharomyces cerevisiae} populations have been shown to clear heteroplasmy within a few generations, with relatively heterogeneous networks and semi-regular spacing of mtDNA \citep{jakubke2021cristae}. This situation corresponds to the `repulsive' version of our model, inducing heteroplasmy variance (helping to clear heteroplasmy) while controlling copy number. In plants, mitochondria normally exist in a fragmented state (low $p$ and $q$) \citep{logan2010dynamic, chustecki2021network}. An intriguing exception to this is the formation of a reticulated network prior to division in the shoot apical meristem -- the tissue that gives rise to the aboveground germline \citep{segui2009mitochondrial}. This formation could be the signature of network structure being employed to shape mtDNA prior to inheritance -- although this employment is also likely to involve facilitating recombination \citep{edwards2021avoiding}. 
 
One technical observation from our work is that the Taylor expansion method for approximating heteroplasmy variance, often employed in mtDNA models \citep{johnston2015stochastic, hoitzing2017stochastic}, has several shortcomings in the face of network structure and other physical phenomenology that induce correlations between mtDNA types. Higher-order terms in the expansion do not immediately fix the discrepancies from simulation; we conclude that the series is likely slow to converge in these cases. We advocate the use of analytic expressions (like our exhaustive sum over states) and explicit simulation to check the validity of such results in future contexts.

Our observations on how network structure influences genetic population structure stand in parallel with the many other phenomena associated with the physical and genetic behaviour of mitochondria. Mitochondrial network structure and dynamics likely fulfil many purposes \citep{hoitzing2015function}, including contributing to mtDNA quality control \citep{twig2008mitochondrial, twig2008fission} via facilitating selection. Here, we assume that selection occurs (if at all) between cell divisions, focussing rather on the behaviour at cell divisions. Previous modelling work has demonstrated the capacity of mitochondrial network structure to shape mtDNA genetics through ongoing processes through the cell cycle \citep{aryaman2019mitochondrial, edwards2021avoiding}; other work has considered the behaviour of controlled mtDNA populations across divisions without considering how that control may be physically manifest \citep{johnston2015closed, johnston2016evolution}. We hope that our models here help bridge the gap between these pictures of spatial dynamics between, and well-mixed behaviour at, cell divisions across a range of eukaryotic life.
 
\section{Methods}
\subsection*{Network simulation}

We constructed a random network via an elongation and branching process; network segments elongated deterministically with rate $e = 0.01$ and branched according to Poissonian dynamics with a given rate $\lambda=0.02$, and terminated if they hit the cell boundary. Network growth was initiated at a number of evenly-spaced seed points that grew initial segments perpendicular to the perimeter of a circular 2D cell, represented by the unit disc, and proceeded until a predetermined network mass had been created; if all segments terminated before this mass was reached, we re-seeded the perimeter, and continued the growth process. By changing the number of seed points from which segments grow, we tune the uniformity of the network structure: a high number of seed points yielded homogeneous network structures, whereas a low number of seed points often lead to heterogeneous network structures (see Fig. \ref{fig:plotcell}).

Next, we distributed mtDNAs in the cell. We considered two mtDNA types, wildtype $W$ and mutant type $M$, each a predetermined proportion of the mtDNA population, as specified by $h$. Proportions $p$ of wild type mtDNA and $q$ of mutant type mtDNA molecules were then placed within the network according to a particular placement rule (see below). The remaining mtDNA molecules were randomly and uniformly distributed in the cell, modelling presence of fragmented organelles in the cytoplasm. First, we considered random and uniform placement of mtDNA within the network, in which every point of the network was equally like to host an mtDNA molecule. Later we restricted to maintain a minimal inter-mtDNA distance within the network (hence where mtDNA molecules are assumed to repel each other).

Finally, we partition the cell and record the number of wildtype $W$ and mutant $M$ mtDNAs in one daughter, the heteroplasmy $h = m/(w+m)$, as well as the proportion of network mass $u$. The process of cell division was modelled by recording only the network mass and mtDNA content of a \textcolor{red}{fixed circular segment spanning an angle $\phi$.}  We are particularly interested in the heteroplasmy variance $V(h)$ and copy number variance $V(N)$, where $N=W+M$, across many realisations of this system.  

\subsection*{Statistical models of mtDNA copy number and heteroplasmy}
We consider $h = \frac{M}{W+M}$ and $N=W+M$ as our key variables. We assume that the parent cell's heteroplasmy level is $h\in [0,1]$, with a total of $N_0$ mtDNA molecules. Thus there are $hN_0$ mutant molecules, and $(1-h)N_0$ wild type molecules of mtDNA. 

We ignore correlations between daughter cells and focus on a single daughter from a cell division. In the daughter, the copy number variance is
\begin{equation}
    V(N) = V(W)+V(M)+2\mathrm{Cov}(W,M)
\end{equation}
Here $V(W)$ and $V(M)$ are the variances of wild-type and mutant mtDNA, respectively, and $\mathrm{Cov}(W,M)$ is the covariance of $W$ with $M$. The heteroplasmy variance, $V(h)$ does not follow a simple form as it deals with a ratio of random variables. Instead, we use either explicit sums for the moments as in the text:

\begin{equation}
    E(f(h)) = \sum_{W_c = 0}^{w_c} P(W_c) \sum_{M_c = 0}^{m_c} P(M_c) \int_0^1 dU \sum_{W_n = 0}^{w_n} P(W_n|U) \sum_{M_n = 0}^{m_n} P(W_n|U) f\left(\frac{M_n+M_c}{W_n+W_c+M_n+M_c}\right).
\end{equation}

with mean $E(h)$ and variance $V(h) = E(h^2) - E(h)^2$, or a first-order Taylor expansion, finding that
\begin{equation}\label{eq:v1h-gen}
V_1(h) = h_M'^2V(M)+h_W'^2V(W)+2h_M'h_W'\mathrm{Cov}(W,M)
\end{equation}
The prefactors $h_M'$ and $h_W'$ are derivatives of the heteroplasmy level considered as a function of $W$ and $M$, and are model dependent (described below). Dividing $V_1(h)$ by $h(1-h)$, we get the normalized heteroplasmy variance, $V_1'(h)$, which conveniently removes some of the dependence on $h$. We also considered higher order terms in this expansion (see Appendix \ref{app:v2h-gen-moments}).

\subsubsection*{The null hypothesis: no mtDNA placement in network}
As our null hypothesis, we considered a binomial segregation model for mtDNA \citep{johnston2019varied}. In this model, no network structure exists and no active mechanisms contribute to the distributions of mtDNA (of either type) to the daughter cell. Supposing that the cell divides such that the daughter consists of a proportion $p_c$ of the parent cell volume, we supposed
\be\label{eq:model-null}
M\sim \mathrm{Bin}(hN_0,p_c) \text{ and } W\sim\mathrm{Bin}((1-h)N_0,p_c)
\ee
The copy number variance of the daughter is then, from the binomial distribution,
\begin{equation}\label{eq:vn-null}
    V(N)=p_c(1-p_c)N_0
\end{equation}
To find the heteroplasmy level variance, we calculated the variances of $W$ and $M$, and the corresponding derivatives in Eqs. (\ref{eq:null-hW.prime}, \ref{eq:null-hM.prime}); the covariance of $W$ and $M$ is zero in this case. A detailed derivation of $V_1(h)$ is presented in Appendix \ref{app:derivations}, the result of which were
\begin{equation}\label{eq:vh-null}
V'(h)=\frac{1-p_c}{p_c} \frac{1}{N_0}
\end{equation}
This is our null case, with nothing actively influencing the placement of mtDNAs within the cell. If this is the case, apportioning of mtDNA to daughter cells is binomial. The familiar expression of $1/N_0$ is recapitulated for symmetric cell divisions, i.e., with $p_c=1/2$, in which case $V'(h)=1/N_0$.

\subsubsection*{Random mtDNA placement in network}
Following intuition and preliminary observation of our simulations, we model the proportion $U$ of network mass inherited by the smaller daughter as beta distributed with mean $E(U)$ and variance $V(U)$. Expected network inheritance $E(U)$ will simply be $p_c$, the proportion of cell volume inherited; $V(U)$ will depend on the spread of the network through the cell, and constitutes a fit parameter in comparing this statistical model to simulation. Hence $U$ is drawn from the beta distribution, $\mathrm{Beta}(\alpha,\beta)$ with mean $E(U) = \alpha/(\alpha+\beta) = p_c$ and variance $V(U) = \frac{\alpha\beta}{(\alpha+\beta)^2(\alpha+\beta+1)}$.

We now write $W_n, W_c$ respectively for the number of wildtype mtDNAs placed in the network and randomly spread in the cytoplasm, and $M_n, M_c$ likewise for mutant mtDNA. $W_c$ and $M_c$ are assumed to follow the binomial partitioning dynamics above. Assuming that mtDNAs in the network are randomly positioned therein, we draw a $u\sim \mathrm{Beta}(\alpha,\beta)$ to reflect the network proportion inherited by the smaller daughter, and write

\be\label{eq:methods-model}
    \begin{split}
        & W_n \sim \mathrm{Bin}\left(w_n,u\right) \\
        & W_c \sim \mathrm{Bin}\left(w_c,p_c\right) \\
        & M_n \sim \mathrm{Bin}\left(m_n,u\right) \\
        & M_c \sim \mathrm{Bin}\left(m_c,p_c\right)
    \end{split}
\ee
where $w_n = p(1-h)N_0, w_c = (1-p)(1-h)N_0, m_n = qhN_0, m_c = (1-q)hN_0$.

The mean and variance of $N$ are readily derived using the laws of iterated expectation and total variance to account for the compound distribution of mtDNA in the network (Appendix). To estimate heteroplasmy variance, we combine the (co)variances of the different types with their respective prefactor (from Eqs. (\ref{eq:net-hW.prime}, \ref{eq:net-hM.prime}) in Appendix \ref{app:taylor}).

\subsubsection*{Repulsive mtDNA placement in network}
Next, we considered the case where mtDNAs placed in the network are not randomly positioned, but instead experience a repulsive interaction, and thus adopt a more even spacing. Capturing this picture perfectly with a statistical model is challenging; instead, we use the following picture. The proportion of inherited network mass $U$ consists of a finite number of `spaces', each of which can be occupied by at most one mtDNA molecule. Choose a number of spaces to fill, then sample mtDNA molecules from the available pool without replacement, assigning each drawn mtDNA to the next unoccupied network space. In this case, the final population of mtDNAs in the network is described by the hypergeometric distribution:
\be\label{eq:model-repel}
    \begin{split}
        & W_n \sim \mathrm{Hypergeometric}(w_n+m_n,w_n,\lfloor u/l \rfloor) \\
        & W_c \sim \mathrm{Bin}\left(w_c,p_c\right) \\
        & M_n \sim \lfloor u/l \rfloor-W_n \\
        & M_c \sim \mathrm{Bin}\left(m_c,p_c\right)
    \end{split}
\ee

We can once again use the laws of total variance and iterated expectation  (see Appendix \ref{app:model-repel}) to estimate heteroplasmy and copy number behaviour. As previously discussed, this model has several shortcomings and is only expected to match qualitative behaviour (see Appendix \ref{app:model-repel}).

\bibliography{refs.bib}

\clearpage
\newpage

\begin{appendices}
\section{Models of mtDNA copy number variance}
 We consider four state variables describing the mtDNA population of a daughter cell after a mother divides: $W_n, W_c, M_n, M_c$, for the wildtype ($W$) and mutant ($M$) mtDNAs contained in a reticulated mitochondrial network (${}_n$) or in fragmented mitochondrial elements in the cytoplasm (${}_c$). An additional variable $U$ describes the proportion of network mass inherited by the daughter cell.

The mother cell has $N_0$ mtDNAs, a proportion $h$ of which are mutants ($h$ is heteroplasmy). Proportions $p$ and $q$ of the wildtype and mutant mtDNAs are contained in the network; the remainder are in fragments in the cytoplasm. 
We consider a daughter inheriting a proportion $p_c$ of the cytoplasm. The mtDNA copy number of the daughter, $N = W_n+W_c+M_n+M_c$, is the sum of all components. We write $W = W_n + W_c$ and $M = M_n+M_c$. Under the null hypothesis of no network, since $M\sim \mathrm{Bin}(hN_0,p_c)$ and $W\sim\mathrm{Bin}((1-h)N_0,p_c)$, we have $N \sim \mathrm{Bin}(N_0, p_c)$ and 
\begin{equation}\label{eq:app-vn-null}
    \begin{split}
    V(N)&=V(W)+V(M)+2\mathrm{Cov}(W,M)\\
        &=V(W)+V(M)\\
        &=(1-h)N_0p_c(1-p_c)+hN_0p_c(1-p_c)\\
        &=p_c(1-p_c)N_0
    \end{split}
\end{equation}
For the random mtDNA distribution model, using the full model (Eq. \ref{eq:model}), and decomposing into the networked and individual mitochondria, i.e., $N=N_c+N_n$, the law of total variance gives
\begin{equation}\label{eq:app-vn-net}
    \begin{split}
    V(N)&=V(N_c)+V(N_n)\\
         &=V(N_c)+E[V(N_n|u)]+V(E[N_n|u])\\
         &= p_c(1-p_c)(1-\kappa)N_0
          + E[U](1-E[U])\kappa N_0+\kappa N_0(\kappa N_0-1)V(U)\\
         &=p_c(1-p_c)N_0+\kappa N_0(\kappa N_0-1)V(U)
    \end{split}
\end{equation}
Overall, this expression captures copy number variance dynamics across a wide range of parameterisations (see Fig. \ref{fig:app-moments}). For the repulsive mtDNA distribution model (\ref{eq:model-repel}) the corresponding expressions are
\begin{equation}\label{eq:app-vn-net-repel}
    \begin{split}
    V(N)&=V(W)+V(M)+2\mathrm{Cov}(W,M)\\
         &=\left((1-\kappa)-2h(1-h)\frac{pqN_0}{\kappa N_0-1}\right)N_0p_c(1-p_c)
         +\left(\kappa^2-2h(1-h)\frac{pq}{\kappa N_0-1}\right)N_0^2V(U)
    \end{split}
\end{equation}
where $\kappa = p(1-h)+qh$. The expression correctly predicts sub-binomial variance for heterogeneous networks, but fails to capture the structure of genetic bias as well as other qualitative dynamics of the system (see Fig. \ref{fig:app-moments-repel}).

\section{Heteroplasmy level variance}\label{app:vh-est}
The heteroplasmy level is the mutant proportion of the cell, i.e.,
\begin{equation}
    h=\frac{M}{W+M}
\end{equation}
where $W$ and $M$ are the numbers of wild-type and mutant type mtDNA, respectively. By definition,
\begin{equation}\label{eq:app-vhest}
    V(h)=\mathrm{E}\left[(h-\mathrm{E}[h])^2\right]
\end{equation}
and $h=h(M,W)$. As the ratio of random variables, $h$ does not admit as straightforward an analysis as $N$. Using the moment expression (Eq. \ref{eq:vh-estimate-sum}) in the main text we can compute its value (and $V(N)$) for a given system. While these expressions provide a good match with simulations for a wide range of parameterisations (see Fig. \ref{fig:sim-model-compare-10}), they do not allow intuitive understanding of the system. Since we sought a more intuitive analysis, we employ first- and second-order Taylor expansions to generate more tractable estimations focussing on key governing parameters.

\subsection{Taylor expansions for heteroplasmy level variance}\label{app:taylor}
Using a first-order Taylor expansion (as in, for example, Ref. \cite{johnston2015stochastic}), we obtain
\begin{equation*}
    \begin{split}
    V(h) 
 & 
     \simeq  E[(h_W'(W-\mu_W)+h_M'(M-\mu_M))^2]\\
 & = E[(h_W'^2(W-\mu_W)^2+h_M'^2(M-\mu_M)^2
 + 2h_W'h_M'(W-\mu_W)(M-\mu_M))^2]\\
 & = h_W'^2V(W)+h_M'^2V(M)+2h_M'h_W'\mathrm{Cov}(M,W)
   \end{split}
\end{equation*}
where prefactors $h_M'$ and $h_W'$ are the partial derivatives of $h(M,W)$ evaluated at the means of the distribution for $M$ and $W$, $\mu_M=hN_0$ and $\mu_W=(1-h)N_0$. In general, we find
\begin{equation}
h_W'(W,M)=\frac{-M}{(W+M)^2} \text{\ \ and \ \ }
h_M'(W,M)=\frac{1}{W+M}-\frac{M}{(W+M)^2}=\frac{W}{W+M}
\end{equation}
or
\begin{equation}
h_W'(\mu_W,\mu_M)=\frac{-\mu_M}{\mu^2} \text{\ \ and \ \ }
h_M'(\mu_W,\mu_M)=\frac{\mu_W}{\mu^2}
\end{equation}
This approximation is used to derive the variance of the heteroplasmy level in all the scenarios considered, so we give it an subscript of $1$ to show it is a first-order Taylor expansion.
\begin{equation}\label{eq:app-v1h-general}
    V_1(h)\approx h_M'^2V(M) +
h_W'^2V(W)+2h_W'h_M'\mathrm{Cov}(M,W)
\end{equation}
The evaluation of the prefactors are model-specific: Under the null hypothesis, where both $W$ and $M$ are binomially distributed with probability $p_c$ and their respective proportion of the population $N$, $\mu_M=p_chN_0$, $\mu_W=p_c(1-h)N_0$ and $\mu = \mu_M+\mu_W$, these expressions evaluate to
\begin{equation}\label{eq:null-hW.prime}
h_W'=-\frac{p_chN_0}{(p_cN_0)^2}=-\frac{h}{p_cN_0}
\end{equation}
\begin{equation}\label{eq:null-hM.prime}
h_M'=\frac{p_c(1-h)N_0}{(p_cN_0)^2}=\frac{1-h}{p_cN_0}.
\end{equation}
For the network model, whether it is with or without mutual repulsion of mtDNA molecules, we write $\mu_W=\mu_{W_n}+\mu_{W_c}=E[U] w_n +p_cw_c$ and $\mu_M=\mu_{M_n}+\mu_{M_c}=E[U] m_n+p_cm_c$, so $\mu=\mu_W+\mu_M=E[U](w_n+m_n)+p_c(w_c+m_c)$ and
\begin{equation}\label{eq:net-hW.prime}
h_W'=-\frac{m_nE[U]+m_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}
\end{equation}
\begin{equation}\label{eq:net-hM.prime}
h_M' = \frac{w_nE[U]+w_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}
\end{equation}
Note that under the assumption that the network is evenly distributed throughout the cell, i.e., $E[U]=p_c$, we recapitulate the expressions for the null hypothesis pre-factors.
\subsubsection*{Null model (no network structure)}\label{app:derivations}
Under the null hypothesis, independent binomial distributions describe both $M$ and $W$. Combining Eqs. \ref{eq:null-hW.prime}, \ref{eq:null-hM.prime}) with the variances of each specie, we find
\begin{equation*}
    \begin{split}
 V_1(h) = & \left(\frac{1}{p_cN_0}\right)^2(h^2V(W)+(1-h)^2V(M)
            -2h(1-h)\mathrm{Cov}(W,M)\\
        = & \left(\frac{1}{p_cN_0}\right)^2(h^2(1-h)p_c(1-p_c)N_0
            +h(1-h)^2p_c(1-p_c)N_0)\\
        = & \left(\frac{h(1-h)}{p_cN_0}\right)((h(1-p_c)+(1-h)(1-p_c))\\
        = & \left(\frac{h(1-h)}{p_cN_0}\right)(1-p_c)
    \end{split}
\end{equation*}
which, weighted by $h(1-h)$, gives Eq. \ref{eq:vh-null}, i.e., the normalized heteroplasmy variance defined as
\begin{equation}\label{eq:app-vh-null}
V'(h)=\frac{V_1(h)}{h(1-h)}=\frac{1-p_c}{p_c}\frac{1}{N_0}
\end{equation}

\subsubsection*{Network model without repulsion}
We next consider the influence of network structure on mtDNA distributions. All random variables are binomially distributed with their respective proportion of the total mtDNA content of the parent as the population, with $p_c$ or $u$ as the probability for cytoplasmic and networked mtDNAs respectively. Using the law of total variance for $M$ and $W$, we find that
\begin{equation}\label{eq:app-vm-net}
    \begin{split}
   V(M) & = E[V(M_n|U)]+V(E[M_n|u])+V(M_c)\\
        & = E[m_nU(1-U)]+V(m_nU)+V(M_c)\\
        & = m_n(E[U]-(V(U)+E[U]^2))+m_n^2V(U)+(1-q)hNp_c(1-p_c)\\
        & = m_nE[U](1-E[U])+m_cp_c(1-p_c)+m_n(m_n-1)V(U)
    \end{split}
\end{equation}
and
\begin{equation}\label{eq:app-vw-net}
    \begin{split}
   V(W) & = E[V(W_n|u)]+V(E[W_n|u])+V(W_c)\\
        & = E[w_nu(1-u)]+V(w_nu)+V(W_c)\\
        & = w_n(E[U]-(V(U)+E[U]^2))+w_n^2V(U)+w_cp_c(1-p_c)\\
        & = w_nE[U](1-E[U])+w_cp_c(1-p_c)+w_n(w_n-1)V(U)
    \end{split}
\end{equation}
Using the law of total covariance, since cytoplasmic copy numbers are uncorrelated, we find that the covariance of $M$ with $W$ is
\begin{equation}\label{eq:app-cmw-net}
    \begin{split}
  \mathrm{Cov}(M,W) & = \mathrm{Cov}(M_n,W_n) \\
    & = E[\mathrm{Cov}(M_n,W_n|u)]+\mathrm{Cov}(E[M_n|u],E[W_n|u])\\ 
    & = E[E[M_nW_n|u]]-E[E[M_n|u]]E[E[W_n|u]]\\
    & = w_nm_n(E[U^2]-E[U]^2)\\
    & = w_nm_nV(U)
    \end{split}
\end{equation}
Combining (co)variances with the prefactors of Eqs. (\ref{eq:net-hM.prime},\ref{eq:net-hW.prime}), Eq. (\ref{eq:app-v1h-general}) gives
\begin{equation}
    \begin{split}
    V_1(h)= &\left(\frac{m_nE[U]+m_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)^2
       \left(w_nE[U](1-E[U])+w_cp_c(1-p_c)+w_n(w_n-1)V(U)\right)\\
      +&\left(\frac{w_nE[U]+w_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)^2
       (m_nE[U](1-E[U])+m_cp_c(1-p_c)+m_n(m_n-1)V(U))\\
      -&2\left(\frac{m_nE[U]+m_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)\left(\frac{w_nE[U]+w_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)w_nm_nV(U)
    \end{split}
\end{equation}
If we assume that $E[U]=p_c$, for which $w_n+w_c=(1-h)N_0$ and $m_n+m_c=hN_0$, we get a simpler expression,
\begin{equation}
    \begin{split}
    V_1(h)= &\left(\frac{h}{p_cN_0}\right)^2
       \left((1-h)N_0p_c(1-p_c)+w_n(w_n-1)V(U)\right)\\
      +&\left(\frac{(1-h)}{p_cN_0}\right)^2
       (hN_0p_c(1-p_c)+m_n(m_n-1)V(U))\\
      -&2\left(\frac{h}{p_cN_0}\right)\left(\frac{(1-h)}{p_cN_0}\right)w_nm_nV(U)
    \end{split}
\end{equation}
Simplifying and gathering terms, we find that
\begin{equation}
    V_1(h)= \left(\frac{h(1-h)}{p_c^2N_0}\right)p_c(1-p_c)+\left(\frac{V(u)}{p_c^2N_0}\right)h^2(1-h)^2(p-q)+\left(\frac{V(U)}{p_cN_0}\right)h(1-h)(ph+q(1-h))
\end{equation}
Dividing by $h(1-h)$ gives $V_1'(h)$,
\begin{equation}\label{eq:app-v1h-net}
V'_1(h) = \frac{1-p_c}{p_cN_0}
        +\frac{h(1-h)}{p_c^2}(p-q)^2V(U)
        -\frac{V(U)}{p_c^2N_0}(ph+q(1-h))
\end{equation}
which we plot for different values of $p$, $q$, $p_c$, and network parameters, $E[U]$ and $V(U)$. The latter two are used to fit a beta distribution with parameters $\alpha$ and $\beta$, with which we model the process of network segregation when a cell divides.% Setting $p_c=1/2$ gives Eq. \ref{eq:v1h-net}.

Looking to gain insights from Eq. \ref{eq:app-v1h-net}, we write $p = q-\delta$. In this case, supposing that $\delta$ is small, wild-type and mutant mtDNA are treated almost equally by the network, with an almost equal proportion of the two admitted into the network.

\begin{equation}
V'_1(h) = \frac{1-p_c}{p_c N_0}
        +\frac{h(1-h)}{p_c^2}\delta^2V(U)
        -\frac{V(U)}{p_c^2N_0}(p+\delta (1-h))
\end{equation}
It will be seen that there is a quadratic dependence on $\delta$, the difference in inclusion probabilities for the different types of mtDNA. For $\delta\neq 0$, the network is genetically biased towards one of the types, to which there is associated an increase in $V'(h)$. For $\delta=0$, the network is unbiased, giving
\begin{equation}\label{eq:app-v1h-net-unbiased}
V'_1(h) = \frac{1-p_c}{p_c N_0}
        -\frac{V(U)}{p_c^2N_0}p
\end{equation}
When $p_c=1/2$, i.e., when cell division is symmetric, we arrive at Eq. \ref{eq:v1h-net} in the main text. Eq. \ref{eq:app-v1h-net-unbiased} suggests that the network structure provides a negative contribution to $V'(h)$, resulting in the blue streak in the first order Taylor expansion of $V'(h)$, whereas, from the simulations, we expected a small \textit{increase} along the diagonal (shown in Fig. \ref{fig:sim-model-compare}). The second order Taylor expansion corrects this (Fig. \ref{fig:app-vh-stats}), with contributions of third order and higher in p and q (Eq. \ref{eq:app-v2h-net}), but it overcompensates; we do not pursue higher order terms, mostly because they are hard to interpret, and present significant difficulties in calculations. We then asked whether statistical simulations would produce a better fit, and we find that there is good support for this model when mtDNA molecules are randomly distributed throughout the network in our simulations.% (figs. \ref{fig:app-sim-theory} and \ref{fig:app-vhestimate-symm}).

%Note that the prefactor $(\frac{1-p_c}{p_c}-\frac{pV(U)}{p_c^2})$ is positive for all parameterizations considered. Indeed, a variance $V(U)\geq 1/4$ cannot be admitted by a beta distribution with $\alpha,\beta\neq 0$. If $\alpha>>\beta$, the mean is almost 1, and the variance is close to zero because it is also multiplied by a factor of the order $\beta/\alpha$. On the other hand, if $\alpha<<\beta$, the mean is itself very small. Lastly, if $\alpha=\beta$, the mean is $1/2$, and the variance is inversely proportional to $4\cdot(2\alpha+1)>4$, i.e., a variance $V(U)<1/4$, since $\alpha>0$. Only in the limit $\alpha=\beta=0$, equality is achieved, under the rather artificial parameterization with $\alpha =\beta = 0$ \citep{johnson1995continuous}. To contrast with our simulations, the largest variance we acquired was $.023$, about a tenth of the approach to this limit, with 4 seed points and $p_c=0.5$. For larger seed numbers, and smaller proportions, the network variance decreases.

\subsubsection*{Network model with repulsion}\label{app:model-repel}
\textcolor{red}{Next we considered networks in which mtDNA molecules within the network were mutually repulsed by each other, setting a minimum distance between mtDNA molecules in the network. We again decompose} $W$ and $M$ into their cytoplasmic and network components, i.e.,
\begin{equation*}
    \begin{split}
        & W = W_n + W_c\\
        & M = M_n + M_c
    \end{split}
\end{equation*}
\textcolor{red}{To assess the effect of mutual self-repulsion of mtDNA, we assumed a model of mtDNA transmission from a parent to its smaller daughter in which we consider the network to be divided into $\lfloor u/l \rfloor$ different compartments. Into each of these compartments a single mtDNA molecule will be placed -- hence $l$ acts to enforce a minimum inter-mtDNA distance due to the repulsion of mtDNA molecules. We then fill these places by sampling, without replacement, mtDNA molecules from the set contained in the network. This corresponds to a hypergeometric model of $\lfloor u/l \rfloor$ samples from a population of $w_n+m_n$ mtDNAs, $w_n$ of which are wild-type:
\be\label{eq:app-model-repel}
    \begin{split}
        & W_n \sim \mathrm{Hypergeometric}(w_n+m_n,w_n,\lfloor u/l \rfloor) \\
        & W_c \sim \mathrm{Bin}\left(w_c,p_c\right) \\
        & M_n \sim \lfloor u/l \rfloor-W_n \\
        & M_c \sim \mathrm{Bin}\left(m_c,p_c\right)
    \end{split}
\ee
where $u\sim\mathrm{beta}(\alpha,\beta)$. The problem with this model is that, to keep the values of $W_n$ and $M_n$ consistent, we must assume that there are exactly $w_n+m_n$ spaces in the network; hence $l=(w_n+m_n)^{-1}$.  In our physical simulations, $l$ is instead set to a distance ($l = 0.01$) that enforces some separation between mtDNAs while making it possible to populate the network through random positions in reasonable time. There are therefore more `spaces' in the simulation than captured by the model, meaning that an even physical spread is less enforced in the simulation than in the model, and the range of variance values supported will be more limited in the simulation.

Accepting this limitation, the variance of $W_n$ is derived using the law of total variance, giving
\begin{equation*}
    \begin{split}
    V(W_n)=&E(V(W_n|U))+V(E(W_n|U))\\
    =&E\left(\lfloor U/l \rfloor\frac{w_n}{w_n+m_n}\frac{w_n+m_n-w_n}{w_n+m_n}\frac{w_n+m_n-\lfloor U/l \rfloor}{w_n+m_n-1}\right)+V\left(\lfloor U/l \rfloor\frac{w_n}{w_n+m_n}\right)\\
    =&\frac{w_nm_n}{l^2(w_n+m_n)^2}\frac{1}{w_n+m_n-1}\{(w_n+m_n)lE[U]-E[U^2]\}+\frac{w_n^2}{l^2(w_n+m_n)^2}V(U)
    \end{split}
\end{equation*}
%Although $l=0.01$ is a constant in our simulations, in our model, we set $l=(w_n+m_n)^{-1}$, so that the network is partitioned to accommodate all mtDNA molecules that need to go in it. That is, only for $w_n+m_n=N_0$ do we partition the model network into as many compartments as there are in the simulated networks, a considerable deviation from the simulation, depending on the model parameterisation in question.

And using the $l=(w_n+m_n)^{-1}$ assumption, 
\begin{equation}
    V(W_n) = w_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])
\end{equation}
For $V(M_n)$ we find
\begin{equation*}
    V(M_n)=\frac{1}{l^2}V(U)+V(W_n)-2\mathrm{Cov}(\lfloor U/l \rfloor,W_n)
\end{equation*}
Using the law of total covariance,
\begin{equation*}
\begin{split}
    \mathrm{Cov}(\lfloor U/l \rfloor,W_n)
    &=\frac{1}{l}\left(E[\mathrm{Cov}(U,W_n|U)]+\mathrm{Cov}(E[U|U],E[W_n|U])\right)\\
    &=\frac{w_n}{l(w_n+m_n)}\left(0+\frac{1}{l}\mathrm{Cov}(U,U)\right)
\end{split}
\end{equation*}
Setting $l=(w_n+m_n)^{-1}$, we find
\begin{equation}
    V(M_n) = (w_n+m_n)^2V(U)+V(W_n)-2(w_n+m_n)w_nV(U)
\end{equation}
respectively. Combining with the variances of the cytoplasmic mtDNA content, we find that
\begin{equation}\label{eq:app-vm-repel}
V(M) = m_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])+p_c(1-p_c)m_c
\end{equation}
\begin{equation}\label{eq:app-vw-repel}
V(W) = w_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])+p_c(1-p_c)w_c.
\end{equation}
As before, $W_n$ has non-zero covariance with $M_n$, due to their mutual dependence on network structure, but no cytoplasmic component covaries with any other component. The overall mutant-wildtype covariance is therefore
\begin{equation*}
\begin{split}
    \mathrm{Cov}(M,W) & =\mathrm{Cov}(M_n,W_n)\\
        & = E[M_nW_n|u]-E[M_n|u]E[W_n|u]\\
        & = w_nm_nE[U^2]-w_nm_nE[U]^2\\
        & = w_nm_nV(U)
    \end{split}
\end{equation*}
In this case, $V_1(h)$, the first-order Taylor expansion of heteroplasmy variance is
\begin{equation}
    \begin{split}
      V_1(h)= &\left(\frac{m_nE[U]+m_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)^2
       \left(w_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])+p_c(1-p_c)w_c\right)\\
      +&\left(\frac{w_nE[U]+w_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)^2
       (m_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])+p_c(1-p_c)m_c)\\
      -&2\left(\frac{m_nE[U]+m_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)\left(\frac{w_nE[U]+w_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\right)w_nm_nV(U)
    \end{split}
\end{equation}
Assuming $E[u]=p_c$, we find that
\begin{equation}
    \begin{split}
      V_1(h)= &\left(\frac{h}{p_cN_0}\right)^2
       \left(w_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])+p_c(1-p_c)w_c\right)\\
      +&\left(\frac{(1-h)}{p_cN_0}\right)^2
       (m_n^2V(U)+\frac{w_nm_n}{w_n+m_n-1}(E[U]-E[U^2])+p_c(1-p_c)m_c)\\
      -&2\left(\frac{h}{p_cN_0}\right)\left(\frac{(1-h)}{p_cN_0}\right)w_nm_nV(U)
    \end{split}
\end{equation}
Gathering some terms and dividing by $h(1-h)$, we find
\begin{equation}\label{eq:v1h-net-repel}
    \begin{split}
      V_1'(h) & = \frac{V(U)}{p_c^2}\left(h(1-h)(p-q)^2-\frac{pq}{\kappa N_0-1}((1-h)^2+h^2)\right)\\
      & + \frac{1-p_c}{p_c N_0}\left(h(1-p)+(1-h)(1-q)\right)\\
      & + \frac{1-p_c}{p_c}\frac{pq}{\kappa N_0-1}\left((1-h)^2+h^2\right)
    \end{split}
\end{equation}
where $\kappa = p(1-h)+qh$.
As before, we plot $V_1'(h)$ for different values of $p$, $q$, $p_c$, and given network parameters $E[U]$ and $V(U)$, used to fit a beta distribution with parameters $\alpha$ and $\beta$, with which we model the process of mtDNA distribution when a cell divides. Fig. \ref{fig:app-sim-model-compare-repel} shows the result of plotting $V_1'(h)$ for networked distributions with mutually repulsive mtDNA molecules. Despite imperfections (Fig. \ref{fig:app-moments-repel}), it will be seen that the the model captures qualitative behaviour of the simulations. %For instance, the maximal increase in variability is smaller, and the copy number variance is smaller than binomial for homogeneous networks, when compared to random reticulated mtDNA distributions.

\begin{figure}
\centering
\includegraphics[width=\linewidth]{symm-50-both-stats-taylor-repel.png}
\caption{\textbf{Approximate model predictions for repulsive interactions between mtDNAs in the network.} $V'(h)$ (left column) and $V(N)$ (right column) under an approximate model for repulsive mtDNA placement in the network.  Qualitatively trends in behaviour are captured, but the magnitudes of the effects involved differ from simulated results (see text).}\label{fig:app-sim-model-compare-repel}
\end{figure}

\subsection{Higher-order moments and second-order Taylor expansion}\label{app:v2h-gen-moments}
Given some observed shortcomings in the ability of the first-order Taylor expansion to capture heteroplasmy variance, we asked whether the next-order terms in the Taylor expansion could refine the estimates. The second-order Taylor expansion of heteroplasmy level variance used for the nonuniform distribution mtDNAs can be expressed as $V_1(h)+V_2(h)$:
\begin{equation}\label{eq:app-v2h-net}
    \begin{split}
        V_2(h) & =  h_M'h_{MM}''\mu_{M,3} + h_W'h_{WW}''\mu_{W,3}\\
       & + (2h_W'h_{MW}''+h_M'h_{WW}'')\mathrm{Cov}(M,W^2) + (2h_M'h_{MW}''+h_W'h_{MM}'')\mathrm{Cov}(M^2,W)\\
       & + \frac{1}{4}h_{MM}''^2\mu_{M,4}
         +\frac{1}{4}h_{WW}''^2\mu_{W,4}\\
       & + (h_{MW}''^2+\frac{1}{2}h_{MM}''h_{WW}'')\mathrm{Cov}(M^2,W^2)\\
       & + h_{MW}''h_{WW}''\mathrm{Cov}(M,W^3)+h_{MW}''h_{MM}''\mathrm{Cov}(M^3,W)
    \end{split}
\end{equation}
where $V_1(h)$ is given by Eq. \ref{eq:v1h-net}, and the derivatives are given by
\begin{equation*}
    \begin{split}
        & h_W' =-\frac{m_nE[U]+m_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\\
        & h_M' = \frac{w_nE[U]+w_cp_c}{(E[U](w_n+m_n)+p_c(w_c+m_c))^2}\\
        & h_{MM}'' = -2W/N^3=-\frac{2(w_nE[U]+w_cp_c)}{((w_n+m_n)E[U]+(w_c+m_c)p_c)^3}\\
        & h_{WW}'' = \frac{2(m_nE[U]+m_cp_c)}{((w_n+m_n)E[U]+(w_c+m_c)p_c)^3}\\
        & h_{MW}'' = (M-W)/N^3=\frac{(m_n-w_n)E[U]+(m_c-w_c)p_c}{((w_n+m_n)E[U]+(w_c+m_c)p_c)^3}
    \end{split}
\end{equation*}

\subsubsection{Higher-order moments from binomial and beta-binomial distributions}
Expanding the Taylor expansion to second-order, we need a number of higher-order moments of the distributions of $W$ and $M$. We start by calculating the third and fourth central moments of $W$ and $M$. For the third order moments, we write
\begin{equation}\label{eq:thirdcentralW}
    \begin{split}
        \mu_{W,3}&=E[((W_n-\mu_{W_n})+(W_c-\mu_{W_c}))^3]\\
        &=E[(W_n-\mu_{W_n})^3+3(W_n-\mu_{W_n})^2(W_c-\mu_{W_c})+
          3(W_n-\mu_{W_n})(W_c-\mu_{W_c})^2+(W_c-\mu_{W_c})^3]\\
          &=\mu_{W_n,3}+\mu_{W_c,3}+3\mathrm{Cov}(W_n^2,W_c)+
            3\mathrm{Cov}(W_n,W_c^2)\\
          &=\mu_{W_n,3}+\mu_{W_c,3},
    \end{split}
\end{equation}
where the final line follows because networked and cytoplasmic mtDNA counts are uncorrelated. As $W_n$ is beta-binomial, we \textcolor{red}{derive} expressions for the third moments of the beta-binomial distribution \textcolor{red}{using Mathematica}; for $W_c$ we use established expressions for the binomial distribution \citep{knoblauch2008closed}:

\begin{equation}
    \begin{split}
    \mu_{W_n,3}&=\frac{w_n\alpha(\beta-\alpha)\beta(2w_n^2+3w_n(\alpha+\beta)+(\alpha+\beta)^2}{(\alpha+\beta)^3(1+\alpha+\beta)(2+\alpha+\beta)}
    \end{split}
\end{equation}
\begin{equation}
    \mu_{W_c,3}=w_c(p_c-3p_c^2+2p_c^3)
\end{equation}
The fourth central moment of $W_n$ is taken from the beta-binomial distribution:
\begin{equation}
    \mu_{W_n,4}=\frac{\alpha\beta w_n(A+B+C+D)}{(\alpha +\beta )^4 (\alpha +\beta +1) (\alpha +\beta +2) (\alpha +\beta +3)}
\end{equation}
where
\begin{equation*}
    A = (\alpha +\beta )^3 \left(\alpha ^2-\alpha  (4 \beta +1)+(\beta -1) \beta \right)
\end{equation*}
\begin{equation*}
    B = 3w_n^3\left(\alpha ^2 (\beta +2)+\alpha  (\beta -2) \beta +2 \beta ^2\right)
\end{equation*}
\begin{equation*}
    C = 6w_n^2\left(\alpha ^3 (\beta +2)+2 \alpha ^2 \beta ^2+\alpha  \beta ^3+2 \beta ^3\right)
\end{equation*}
and
\begin{equation*}
    D = w_n(\alpha +\beta )^2(\alpha ^2 (3 \beta +7)+\alpha  (3 \beta ^2-10 \beta -1)+\beta  (7 \beta -1)
\end{equation*}
The fourth central moment of $W_c$ is from the binomial distribution:
\begin{equation}
    \mu_{W_c,4}=w_cp_c(1-p_c)(1+(3w_c-6)p_c(1-p_c)).
\end{equation}
For $M$ we find the same expression, but with different prefactors
\begin{equation}
    \mu_{M_n,3}=\frac{m_n\alpha(\beta-\alpha)\beta(2m_n^2+3m_n(\alpha+\beta)+(\alpha+\beta)^2}{(\alpha+\beta)^3(1+\alpha+\beta)(2+\alpha+\beta)}
\end{equation}
\begin{equation}
    \mu_{M_c,3}=m_c(p_c-3p_c^2+2p_c^3)
\end{equation}

For terms in $M$ we follow the same approach of recruiting central moment results from the beta-binomial and binomial distributions. The same expression structures arise, but with different prefactors reflecting the mutant population:

\begin{equation}
    \mu_{M_n,3}=\frac{m_n\alpha(\beta-\alpha)\beta(2m_n^2+3m_n(\alpha+\beta)+(\alpha+\beta)^2}{(\alpha+\beta)^3(1+\alpha+\beta)(2+\alpha+\beta)}
\end{equation}
\begin{equation}
    \mu_{M_c,3}=m_c(p_c-3p_c^2+2p_c^3)
\end{equation}
and
\begin{equation}
    \mu_{M_n,4}=\frac{\alpha\beta m_n(A+B+C+D)}{(\alpha +\beta )^4 (\alpha +\beta +1) (\alpha +\beta +2) (\alpha +\beta +3)}
\end{equation}
where
\begin{equation*}
    A = (\alpha +\beta )^3 \left(\alpha ^2-\alpha  (4 \beta +1)+(\beta -1) \beta \right)
\end{equation*}
\begin{equation*}
    B = 3m_n^3\left(\alpha ^2 (\beta +2)+\alpha  (\beta -2) \beta +2 \beta ^2\right)
\end{equation*}
\begin{equation*}
    C = 6m_n^2\left(\alpha ^3 (\beta +2)+2 \alpha ^2 \beta ^2+\alpha  \beta ^3+2 \beta ^3\right)
\end{equation*}
and
\begin{equation*}
    D = m_n(\alpha +\beta )^2(\alpha ^2 (3 \beta +7)+\alpha  (3 \beta ^2-10 \beta -1)+\beta  (7 \beta -1)
\end{equation*}
The fourth central moment of $M_c$ is
\begin{equation}
    \mu_{M_c,4}=m_cp_c(1-p_c)(1+(3m_c-6)p_c(1-p_c))
\end{equation}

\subsubsection{Covariance calculations}
To calculate the covariances, we use the law of total covariance, which for RVs $X,Y$ and $Z$ states that
\begin{equation*}
    \mathrm{Cov}(X,Y)=E[\mathrm{Cov}(X,Y|Z)]+\mathrm{Cov}(E[X|Z],E[Y|Z])
\end{equation*}
Using the identity $\mathrm{Cov}(X,Y)=E[XY]-E[X]E[Y]$, we find we retain the terms
\begin{equation*}
    \mathrm{Cov}(X,Y)=E[E[XY|Z]]-E[E[X|Z]]E[E[Y|Z]]
\end{equation*}
In this case, when $u$ is fixed, the variables $W$ and $M$ are independent RVs, so the first term is the $E[E[X|Z]E[Y|Z]]$. Using these findings, the necessary covariances of higher order in the RVs $M$ and $W$ are
\begin{equation}
    \begin{split}
    \mathrm{Cov}(W^2,M)&=\mathrm{Cov}(W_n^2,M_n)+
            2E[W_c]\mathrm{Cov}(W_nW_c,M_n)\\
        &=E[E[W_n^2|u]E[M_n|u]]-E[E[W_n^2|u]]E[E[M_n|u]]+2E[W_c]\mathrm{Cov}(W_n,M_n)\\
        &=E[(w_nm_nu^2+w_nm_n(w_n-1)u^3]-E[w_nu+w_n(w_n-1)u^2]E[m_nu]\\
        &\igjindent +2p_cw_cw_nm_nV(U)\\
        &=w_nm_n\left(V(U)+(w_n-1)(E[U^3]-E[U]E[U^2]\right)+2p_cw_cw_nm_nV(U)
    \end{split}
\end{equation}
Note that we have used that $E[W_n^2|u]=V(W_n|u)+E[W_n|u]^2=w_nu+w_n(w_n-1)u^2$. To calculate the covariance of $W^2$ with $M^2$, we also need $E[M_n^2|u]=m_nu+m_n(m_n-1)u^2$
\begin{equation}
    \begin{split}
    \mathrm{Cov}(W^2,M^2)&=\mathrm{Cov}(W_n^2,M_n^2)+
            2\mathrm{Cov}(W_n^2,M_nM_c)\\
            &\igjindent +2\mathrm{Cov}(W_nW_c,M_n^2)
             +4\mathrm{Cov}(W_nW_c,M_nM_c)\\
    &=w_nm_nE[(u+(w_n-1)u^2)(u+(m_n-1)u^2)]\\
    &\igjindent -w_nm_nE[U+(w_n-1)u^2]E[U+(m_n-1)u^2]\\
    &=w_nm_n(E[U^2]+(w_n+m_n-2)E[U^3]+(w_n-1)(m_n-1)E[U^4])\\
    &\igjindent -w_nm_n(E[U]^2+(w_n+m_n-2)E[U]E[U^2]+(w_n-1)(m_n-1)E[U^2]^2)\\
    &\igjindent +2p_cw_cw_nm_n\left(V(U)+(m_n-1)(E[U^3]-E[U]E[U^2])\right)\\
    &\igjindent +2p_cm_cw_nm_n\left(V(U)+(w_n-1)(E[U^3]-E[U]E[U^2])\right)\\
    &\igjindent +4p_c^2w_cm_cw_nm_nV(U)\\
    &=w_nm_n(V(U)+(w_n+m_n-2)(E[U^3]-E[U]E[U^2])+(w_n-1)(m_n-1)(E[U^4]-E[U^2]^2))\\
    &\igjindent +2p_cw_cw_nm_n\left(V(U)+(m_n-1)(E[U^3]-E[U]E[U^2])\right)\\
    &\igjindent +2p_cm_cw_nm_n\left(V(U)+(w_n-1)(E[U^3]-E[U]E[U^2])\right)\\
    &\igjindent +4p_c^2w_cm_cw_nm_nV(U)
    \end{split}
\end{equation}
\begin{equation}
    \begin{split}
        \mathrm{Cov}(W^3,M)&=\mathrm{Cov}(W_n^3,M_n)+ 3\mathrm{Cov}(W_n^2W_c,M_n)+3\mathrm{Cov}(W_nW_c^2,M_n)\\
        &=E[E[W_n^3|u]E[M_n|u]]-E[E[W_n^3|u]]E[E[M_n|u]]\\
        &\igjindent +3E[W_c]\mathrm{Cov}(W_n^2,M_n)+3E[W_c^2]\mathrm{Cov}(W_n,M_n)\\
        &=E[\mu_3'(W_n|u)E[M_n|u]]-E[\mu_3'(W_n|u)]E[E[M_n|u]]\\
        &\igjindent +3E[W_c]\mathrm{Cov}(W_n^2,M_n)+3E[W_c^2]\mathrm{Cov}(W_n,M_n)\\
        &=w_nm_n(E[U^2]+3(w_n-1)E[U^3]+(w_n^2-3w_n+2)E[U^4])\\
        &\igjindent -w_nm_n(E[U]^2+3(w_n-1)E[U]E[U^2]+(w_n^2-3w_n+2)E[U]E[U^3])\\
    &\igjindent +3p_cw_cw_nm_n\left(V(U)+(w_n-1)(E[U^3]-E[U]E[U^2]\right)\\
    &\igjindent +3(w_cp_c(1-p_c)+w_c^2p_c^2)w_nm_nV(U)\\
    &=w_nm_n(V(U)+3(w_n-1)\left(E[U^3]-E[U]E[U^2]\right)+(w_n^2-3w_n+2)\left(E[U^4]-E[U]E[U^3]\right))\\
    &\igjindent +3p_cw_cw_nm_n\left(V(U)+(w_n-1)(E[U^3]-E[U]E[U^2]\right)\\
    &\igjindent +3(w_cp_c(1-p_c)+w_c^2p_c^2)w_nm_nV(U)
    \end{split}
\end{equation}
Here we have used $E[W_c^2]=V(W_c)+E[W_c]^2=w_cp_c(1-p_c)+w_c^2p_c^2$ and, since the third non-central moment $\mu_3'(W_n|u)$ expressed in terms of the third central moment $\mu_3(W_n|u)$ is $\mu_3(W_n|u)+3\mu E[W_n^2|u]+E[W_n|u]^3$, where $\mu_3(W_n|u)=w_n(u-3u^2+2u^3)$, we find that
\begin{equation*}
    \mu_3'(W_n|u)=w_n((w_n^2-3w_n+2)u^3+3(w_n-1)u^2+u)
\end{equation*}
Lastly, $\mathrm{Cov}(W,M^2)$ and  by the symmetry of the problem
\begin{equation}
    \begin{split}
    \mathrm{Cov}(W,M^2)&=w_nm_n\left(V(U)+(m_n-1)(E[U^3]-E[U]E[U^2]\right)+2p_cm_cw_nm_nV(U)
    \end{split}
\end{equation}
\begin{equation}
    \begin{split}
        \mathrm{Cov}(W,M^3)&=w_nm_n(V(U)+3(m_n-1)\left(E[U^3]+E[U]E[U^2]\right)+(m_n^2-3m_n+2)\left(E[U^4]-E[U]E[U^3]\right))\\
    &+3p_cm_cw_nm_n\left(V(U)+(m_n-1)(E[U^3]-E[U]E[U^2]\right)\\
    &+3(m_cp_c(1-p_c)+m_c^2p_c^2)w_nm_nV(U)
    \end{split}
\end{equation}
In Figs. \ref{fig:app-moments}-\ref{fig:app-moments-repel}, we plot these expressions for the various moments and covariances in the system compared to those arising from our simulation model. We generally observe good agreement between theory and simulation (most departures are on a very small scale compared to the overall scale of the corresponding expression; arising due to small deviations from the beta-distribution model for network mass). However, the overall second-order Taylor expression still deviates substantially from observed heteroplasmy variance (Fig. \ref{fig:app-vh-stats}). One aspect of the first-order model is improved -- the increase on the $p=q$ diagonal. But the off-diagonal behaviour is substantially compromised, suggesting an overcompensation to the errors in the previous order. We conclude that higher-still terms in the expansion will be required to more perfectly capture the behaviour, and that convergence to the true behaviour may be rather slow.

%\subsubsection{Comparison of individual statistics}
Here, we present the comparisons of simulation results with model results in both models to all relevant orders. First, one should note that the second order result only applies to the models with random mtDNA placement in the network, and then only for the heteroplasmy variance. This is because $h$ is a ratio of random variables, and which is differentiable an arbitrary number of times with respect to both variables, $W$ and $M$; the copy number variance, however, is linear in these random variables, and so the approximation is the same for all orders starting at first. Fig. \ref{fig:app-vh-stats} shows comparisons of simulation results (top row) with first and second order results (middle and bottom rows), respectively. Here we see clearly that both first and second order approximations were needed to capture the behavior displayed in our simulations, but that neither provides a reasonable match: the first order approximation departs significantly along the diagonal, displaying a small decrease as opposed to a small increase; the second order approximation massively overestimates, causing a far too large an increase along the diagonal.

\begin{figure}[!ht]
        \centering \includegraphics[width = 0.85\linewidth]{symm-50-vh-stats-no-repel.png}
        \caption{\textbf{Comparison of simulation results with analytic model results for first and second order Taylor expansion.} By row,  simulation, first, and second order analytic results for $V'(h)$ for random placement of mtDNAs in the network. The first order theory in the second row produces results similar to our simulation results on the off-diagonal, but fails to reproduce the increase observed along the diagonal. The second order theory, while loosely retaining the same structure on the off-diagonal as the first order theory, overestimates this increase along the diagonal. We expect that our model would captures this behavior if we were to derive even higher order terms.}\label{fig:app-vh-stats}
\end{figure}
%Figure \ref{fig:app-vn-stats} compares \textit{in silico} model results with theory of $V(N)$ for random mtDNA distributions in the network. For networks with large variance (left; small seed numbers), the mtDNA copy number increases rapidly as the inclusion probabilities increase; the variance is maximized when all mtDNAs are in the network.

%Fig. \ref{fig:app-vh-stats-repel} compares simulation results with first order model results in symmetric cell divisions with parent cell heteroplasmy level $h_0=0.5$. The repulsive model for mtDNAs in the network displays very little increase along the diagonal, with smaller increases to $V'(h)$ than for random mtDNA distributions. in the network. When homogeneous, however, the networks have the effect of decreasing the heteroplasmy variance for maximal genetic bias. Our repulsive mtDNA model again captures the behavior for large genetic bias, but along the diagonal (no genetic bias), it decreases substantially, for reasons discussed in the text.

%Figure \ref{fig:app-vn-stats-repel} compares \textit{in silico} model results with theory of $V(N)$ for repulsive reticulated mtDNAs. For networks with large variance (left; small seed numbers), the mtDNA copy number increases rapidly as the inclusion probabilities increase, and plateus; our model indicates that $V(N)$ continues to increase beyond this plateau. For intermediate variance (middle; intermediate seed numbers), the discrepancy between the results appear the largest. Here the analytic results do not capture the spuriousness of the relationship between inclusion probabilities and $V(N)$. For uniformly distributed networks (right; large seed numbers), we see that the mutual repulsion of mtDNA causes $V(N)$ to drop below the null value of 25, indicating finer-than-binomial control of mtDNA copy number.

% Moments
\begin{figure}
        \centering \includegraphics[width = 0.85\linewidth]{all-props-moments-no-repel.png}
        \caption{\textbf{Comparisons of individual simulation moments (vertical axes) and analytic moments (horizontal axes) for random placement of mtDNA in the network (Eq. \ref{eq:model}) for varying proportions of cell volume inherited by the smallest daughter.} Colors reflect the proportion of parent cell volume apportioned to the daughter of interest.  }\label{fig:app-moments}
\end{figure}
\begin{figure}
        \centering \includegraphics[width = 0.85\linewidth]{all-props-moments-repel.png}
        \caption{\textbf{Comparisons of individual simulation moments (vertical axes) and analytic moments (horizontal axes) for repulsive placement of mtDNA in the network (Eq. \ref{eq:model-repel}) for varying parent cell cytoplasm proportions inherited by the smallest daughter.}  Colors reflect the proportion of parent cell volume apportioned to the daughter of interest.
        }\label{fig:app-moments-repel}
\end{figure}
\end{appendices}
\end{document}
